package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Remote new tab (third version)",
   "main": "src/main.js",
   "scripts": {
-    "setup": "npm run clean && npm run build:static",
+    "setup": "npm run clean && npm run build:static && npm run build:offline",
```

Comment:
hmmm, this is true. I thought it was looking at `src/static`. Maybe I should change it to that? I need the file to be there during dev.